2024-09-12 22:00:28,567 INFO    MainThread:3858502 [wandb_setup.py:_flush():77] Current SDK version is 0.17.8
2024-09-12 22:00:28,567 INFO    MainThread:3858502 [wandb_setup.py:_flush():77] Configure stats pid to 3858502
2024-09-12 22:00:28,567 INFO    MainThread:3858502 [wandb_setup.py:_flush():77] Loading settings from /home/ray/.config/wandb/settings
2024-09-12 22:00:28,567 INFO    MainThread:3858502 [wandb_setup.py:_flush():77] Loading settings from /home/ray/mnt/cluster_storage/ai_geolocation/geo-clip/geoclip/wandb/settings
2024-09-12 22:00:28,567 WARNING MainThread:3858502 [wandb_setup.py:_flush():77] Unknown environment variable: WANDB_POPULATE_RUN_LOCATION_HOOK
2024-09-12 22:00:28,567 WARNING MainThread:3858502 [wandb_setup.py:_flush():77] Unknown environment variable: WANDB_SETUP_API_KEY_HOOK
2024-09-12 22:00:28,567 WARNING MainThread:3858502 [wandb_setup.py:_flush():77] Unknown environment variable: WANDB_PROCESS_RUN_INFO_HOOK
2024-09-12 22:00:28,567 INFO    MainThread:3858502 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-09-12 22:00:28,567 INFO    MainThread:3858502 [wandb_setup.py:_flush():77] Applying setup settings: {'_disable_service': False}
2024-09-12 22:00:28,567 INFO    MainThread:3858502 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-12 22:00:28,567 INFO    MainThread:3858502 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-09-12 22:00:28,568 INFO    MainThread:3858502 [wandb_init.py:_log_setup():524] Logging user logs to /home/ray/mnt/cluster_storage/ai_geolocation/geo-clip/geoclip/wandb/run-20240912_220028-srmicbr0/logs/debug.log
2024-09-12 22:00:28,568 INFO    MainThread:3858502 [wandb_init.py:_log_setup():525] Logging internal logs to /home/ray/mnt/cluster_storage/ai_geolocation/geo-clip/geoclip/wandb/run-20240912_220028-srmicbr0/logs/debug-internal.log
2024-09-12 22:00:28,568 INFO    MainThread:3858502 [wandb_init.py:init():607] calling init triggers
2024-09-12 22:00:28,568 INFO    MainThread:3858502 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {'learning_rate': {0.0001}, 'step_size': {30}, 'epochs': {10}, 'optimizer_gamma': {0.1}}
2024-09-12 22:00:28,568 INFO    MainThread:3858502 [wandb_init.py:init():632] re-initializing run, found existing run on stack: e6s60fss
2024-09-12 22:00:28,571 INFO    MainThread:3858502 [wandb_run.py:_finish():2160] finishing run vannevar/ai-geolocation/e6s60fss
2024-09-12 22:00:28,572 INFO    MainThread:3858502 [jupyter.py:save_history():444] not saving jupyter history
2024-09-12 22:00:28,573 INFO    MainThread:3858502 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-09-12 22:00:28,573 INFO    MainThread:3858502 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-09-12 22:00:28,573 INFO    MainThread:3858502 [wandb_run.py:_atexit_cleanup():2424] got exitcode: 0
2024-09-12 22:00:28,574 INFO    MainThread:3858502 [wandb_run.py:_restore():2406] restore
2024-09-12 22:00:28,575 INFO    MainThread:3858502 [wandb_run.py:_restore():2412] restore done
2024-09-12 22:00:28,575 INFO    MainThread:3858502 [wandb_run.py:_on_finish():2677] communicating current version
2024-09-12 22:00:28,641 INFO    MainThread:3858502 [wandb_run.py:_on_finish():2686] got version response upgrade_message: "wandb version 0.18.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-09-12 22:00:33,489 INFO    MainThread:3858502 [wandb_run.py:_footer_history_summary_info():4078] rendering history
2024-09-12 22:00:33,490 INFO    MainThread:3858502 [wandb_run.py:_footer_history_summary_info():4110] rendering summary
2024-09-12 22:00:33,496 INFO    MainThread:3858502 [wandb_run.py:_footer_sync_info():4037] logging synced files
2024-09-12 22:00:33,597 INFO    MainThread:3858502 [wandb_init.py:init():657] starting backend
2024-09-12 22:00:33,597 INFO    MainThread:3858502 [wandb_init.py:init():661] setting up manager
2024-09-12 22:00:33,601 INFO    MainThread:3858502 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-09-12 22:00:33,601 INFO    MainThread:3858502 [wandb_init.py:init():669] backend started and connected
2024-09-12 22:00:33,604 INFO    MainThread:3858502 [wandb_run.py:_label_probe_notebook():1344] probe notebook
2024-09-12 22:00:33,604 INFO    MainThread:3858502 [wandb_run.py:_label_probe_notebook():1354] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-09-12 22:00:33,604 INFO    MainThread:3858502 [wandb_init.py:init():767] updated telemetry
2024-09-12 22:00:33,615 INFO    MainThread:3858502 [wandb_init.py:init():800] communicating run to backend with 90.0 second timeout
2024-09-12 22:00:34,682 INFO    MainThread:3858502 [wandb_init.py:init():851] starting run threads in backend
2024-09-12 22:00:38,310 INFO    MainThread:3858502 [wandb_run.py:_console_start():2463] atexit reg
2024-09-12 22:00:38,310 INFO    MainThread:3858502 [wandb_run.py:_redirect():2309] redirect: wrap_raw
2024-09-12 22:00:38,311 INFO    MainThread:3858502 [wandb_run.py:_redirect():2374] Wrapping output streams.
2024-09-12 22:00:38,311 INFO    MainThread:3858502 [wandb_run.py:_redirect():2399] Redirects installed.
2024-09-12 22:00:38,312 INFO    MainThread:3858502 [wandb_init.py:init():894] run started, returning control to user process
